RELEASE/scripts/autoscend/auto_cooking.ash

```diff
@@ -611,6 +606,52 @@ void consumeStuff()
 	}
 }
 
+boolean consumeFortune()
+{
+	if (contains_text(get_counters("Fortune Cookie", 0, 200), "Fortune Cookie"))
+	{
+		return false;
+	}
+
+	// Don't get lucky numbers for the first semi-rare if we still need to adventure in the outskirts
+	if (my_turncount() < 80 && internalQuestStatus("questL05Goblin") == 0)
```

Comment:
```suggestion
	if (my_turncount() < 80 && (internalQuestStatus("questL05Goblin") < 1 || item_amount($item[Knob Goblin encryption key] < 1))
```

Can farm the encryption key before you've accepted the quest. I'm not sure if it's disappears when you use it though, if not the quest property test could be dropped from this condition.

Reply:
it disappears